nagios/conf.d/testnet.cfg

```diff
@@ -123,3 +123,13 @@ define service {
 	notification_interval 0
 	check_command return-ok
 }
+
+define service {
```

Comment:
Please keep only rnodes in `{test,dev}net.cfg`. Use `misc.cfg` or anything else for other host and services.